packages/flutter_android_lifecycle/example/ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
nit: I think this `ios/` directory can be removed.

Reply:
Do we know for sure that we can delete it? The android_intent plugin seems to include an /ios directory:
https://github.com/flutter/plugins/tree/master/packages/android_intent